README.md

```diff
@@ -67,7 +67,7 @@ export default App;
 
 ## Running on other platforms
 
-Check the following repository for detailed examples for running the Dock Wallet SDK on NodeJS, Web, and Flutter.
+Check the following repository for detailed examples for running the Truvera Wallet SDK on NodeJS, Web, and Flutter.
 
 https://github.com/docknetwork/wallet-sdk-examples
```

Comment:
this link should probably be wrapped in markdown link syntax too `[https://github.com/docknetwork/wallet-sdk-examples](https://github.com/docknetwork/wallet-sdk-examples)`